app/main/subsystems/json.ts

```diff
@@ -0,0 +1,5 @@
+import { constant } from "fp-ts/lib/function"
+import { pureComponent } from "../lifecycle"
+import { jsonSerializer } from "../../common/serializers"
```

Comment:
Suggestion: I prefer to avoid relative imports that go up and use instead `import {jsonSerializer} from "app/common/serializers"`

Reply:
That's WebStorm rewriting my imports :see_no_evil: